_config.yml

```diff
@@ -25,6 +25,9 @@ contact_phone_number_location: 'bottom' # acceptable values are 'top' and 'botto
 contact_maintenance_start_time: null
 contact_maintenance_end_time: null
 
+# Unplanned outage configuration
+contact_unplanned_outage: null
```

Comment:
If we're using this as a boolean when configuring the outage (i.e. setting to `true`), I think it might be clearer to whomever's tasked with that if the default were `false`.

```suggestion
contact_unplanned_outage: false
```